### PR TITLE
ESLint whole Tact codebase

### LINKED ISSUE #141: Fix issues reported by ESLint

### PR DESCRIPTION
**Pull request by anton-trunov**
Fixes #141

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1,11 @@
+node_modules/
+dist/
+converage/
```
Comment (novusnota) [inline .eslintignore]: `converage/` → `coverage/`

Comment (anton-trunov) [inline .eslintignore]: good catch!
```suggestion
coverage/
```

File: examples/increment.spec.ts
Comment (novusnota) [inline examples/increment.spec.ts:1]: Counters state is now neglected due to `counters` and `counters2` being unused. Is it ok?

Comment (anton-trunov) [reply examples/increment.spec.ts:1]: I think it's ok, I don't see how those vars were used. It's fine to remove that provided we are not after some side effect when calling `contract.getCounters` and `contract.getCounters2`. The test can be improved, though. We should compare the counters against some expected values. Could be implemented in a separate PR

**Review by novusnota — DISMISSED** [review]

**Pull request merged**